client/src/components/chat/Chat.js

```diff
@@ -6,10 +6,12 @@ import { MessagesPanel } from "./MessagesPanel";
 import { io } from "socket.io-client";
 import UserContext from "../../contexts/user/UserContext";
 import Web3Context from "../../contexts/web3/Web3Context";
+import userContext from "../../contexts/user/UserContext";
 const SERVER = "http://localhost:3000";
 
 const Chat = (props) => {
     const { accessToken } = useContext(UserContext);
+    const { organization } = useContext(userContext);

```

Comment:
We have already loaded and importing the userContext on the line before this. Change `{ accessToken }` to `{ accessToken, organization }` to get the context